clients/config/overrides.ts

```diff
@@ -0,0 +1,6 @@
+import { CallOverrides } from "@ethersproject/contracts";
+
+export const defaultGasOverrides: CallOverrides = {
+  maxFeePerGas: 1000000000000,
```

Comment:
This needs to be higher than `maxPriorityFeePerGas`

Reply:
I think this has an extra zero? Need to have 100gwei

Reply:
used config from docs: https://docs.immutable.com/docs/zkEVM/architecture/gas-config